service.autosubs/resources/language/resource.language.en_GB/strings.po

```diff
@@ -0,0 +1,63 @@
+# Autosubs language file
```

Comment:
resources/__init__.py file shouldn't be needed in py3

Reply:
sure, thanks!